nucypher/blockchain/eth/deployers.py

```diff
@@ -31,14 +31,15 @@
     AdjudicatorAgent)
 from nucypher.blockchain.eth.constants import DISPATCHER_CONTRACT_NAME
 from nucypher.blockchain.eth.decorators import validate_secret
-from nucypher.blockchain.eth.interfaces import BlockchainDeployerInterface, BlockchainInterface
-from nucypher.blockchain.eth.registry import AllocationRegistry, EthereumContractRegistry
+from nucypher.blockchain.eth.interfaces import BlockchainDeployerInterface
+from nucypher.blockchain.eth.registry import AllocationRegistry
 
 
 class ContractDeployer:
 
     agency = NotImplemented
     contract_name = NotImplemented
+    num_deployment_steps = NotImplemented
```

Comment:
How about the name -> `number_of_deployment_transactions`?

Reply:
✔️